modules/runner/src/main/java/org/apache/ignite/app/IgniteCliRunner.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.app;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.apache.ignite.internal.app.IgnitionImpl;
+
+/**
+ * The main entry point for run new Ignite node from CLI toolchain.
+ */
+public class IgniteCliRunner {
+    /**
+     * Main method for run new Ignite node.
+     *
+     * @param args CLI args to start new node.
+     * @throws IOException if any issues with reading config file.
+     */
+    public static void main(String[] args) throws IOException {
+        Args parsedArgs = null;
+
+        try {
+            parsedArgs = Args.parseArgs(args);
+        }
+        catch (Args.ParseException e) {
+            System.out.println(e.getMessage());
+            System.exit(1);
```

Comment:
empty line. Also, should we write here some clarification what happened before `e.getMessage()`?

Reply:
It's so tricky to improve the messaging for all types of issues without more complex argument parsing logic. I think that usage message + [some details] (for --config path parsing) is ok for current arguments. WDYT?

Anyway, error reporting updated, you can check the current aprroach.